mtools/test/logfiles/mongod_3_4_2.log

```diff
@@ -0,0 +1,27 @@
+2017-10-09T12:44:35.031+0530 I CONTROL  [initandlisten] MongoDB starting : pid=10686 port=27017 dbpath=/Users/nishant/testcase/data 64-bit host=Nishants-MacBook-Pro.local
```

Comment:
Since this log is intended for testing the connection feature, the naming should reflect that. Maybe `mongod_3_4-2_connection-stats.log`?

Reply:
For completeness can you generate and include test logs for 3.0 and 3.2? I would also suggest latest 3.4.x rather than 3.4.2.